versions.json

```diff
@@ -23,11 +23,6 @@
       "branch": "8.18",
       "previousMajor": true
     },
-    {
```

Comment:
We still want to keep this until 8.19 is released

Reply:
👍 reverted